ui/app/serializers/mfa-method.js

```diff
@@ -0,0 +1,18 @@
+import { assign } from '@ember/polyfills';
+import ApplicationSerializer from './application';
+
+export default class KeymgmtKeySerializer extends ApplicationSerializer {
+  normalizeItems(payload) {
+    if (payload.data.keys && Array.isArray(payload.data.keys)) {
+      let data = payload.data.keys.map((key) => {
+        let model = payload.data.key_info[key];
+        model.id = key;
+        return model;
+      });
+      return data;
+    }
+    assign(payload, payload.data);
```

Comment:
There's a deprecation warning on ember `assign`. You could use `Object.assign` or the spread operator instead.